src/applications/personalization/search-representative/sass/search-representative.scss

```diff
@@ -1,3 +1,15 @@
+.form-review-panel-page:nth-of-type(1) dt {
```

Comment:
We'll have to be careful with this because it looks like it affects the other Review panels

Reply:
Updated this to only target the first dt within the `basicInformation` chapter